packages/fxa-payments-server/src/routes/Subscriptions/SubscriptionItem.tsx

```diff
@@ -44,6 +44,7 @@ export const SubscriptionItem = ({
   const paymentProvider: PaymentProvider | undefined =
     customer?.payment_provider;
   const promotionCode = customerSubscription.promotion_code;
+  const labelId = 'subscription-' + plan?.product_id;
```

Comment:
Looks like this _could_ create `labelId` like `'subscription-undefined'`, but then we exit early below on L49 if the `plan` was falsy. 🎉

Reply:
This SGTM. You could maybe add a comment above the `labelId` declaration to say something like what Peter's got so if that changes in the future folks can see the comment at least. 🙂